view/theme/frio/css/style.css

```diff
@@ -657,6 +661,22 @@ nav.navbar .nav > li > button:focus
     margin: 0px;
     padding: 12px 12px;
 }
+#search-mobile {
+    position: fixed;
+    top: 90px;
+    left: 0;
+    right: 0;
+    z-index: 2;
+
+    background-color: $background_color;
+
+    /* fix bootstrap well not playing well with data-target slide animation */
```

Comment:
```suggestion
    /* fix Bootstrap not playing well with data-target slide animation */
```

Reply:
I changed the comment to `fix bootstrap .well class not playing well with data-target slide animation`
that was awkward!

Reply:
Ha ha ha yes, I thought it was a typo!